source/docs/software/advanced-controls/introduction/control-system-basics.rst

```diff
@@ -33,6 +44,15 @@ The figure below shows a system with a hypothetical input and output. Since the
 .. image:: images/control-system-basics-whatisgain.png
    :alt: A system diagram with hypothetical input and output
 
+What is a Model?
+----------------
+
+A *model* of your mechanism is a set of math equations which describe its expected behavior.
+
+They can be built up from first-principles physics equations, and refined by doing experiments on the mechanism.
```

Comment:
> first-principles physics equations

I'm not sure I'm familiar with the term?

Reply:
"first principles" is another term for a set of axioms - that is, in the sense of a single set of starting principles that is sufficient to derive the whole theory.

Reply:
It's definitely loaded and, while I"ve heard it used a lot in the context to describe "basic hs physics equations", it probably doesn't mean the same to everyone. Dalton had asked it removed earlier and I pulled out all the ones I could find, but missed this one.